packages/sanes-chrome-extension/src/routes/share-identity/index.dom.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { Store } from 'redux';
+import { RootState } from '../../store/reducers';
+import { aNewStore } from '../../store';
+import { travelToShareIdentity } from './test/travelToShareIdentity';
+import {
+  clickOnAcceptButton,
+  clickOnBackButton,
+  clickOnRejectButton,
+  confirmSelectionButton,
+  checkPermanentRejection,
+} from './test/operateShareIdentity';
+
+describe('DOM > Feature > Share Identity', (): void => {
+  let store: Store<RootState>;
+  let identityDOM: React.Component;
+
+  beforeEach(async () => {
+    store = aNewStore();
+    identityDOM = await travelToShareIdentity(store);
+  });
+
+  it('should accept incoming request and come back', async (): Promise<void> => {
+    await clickOnAcceptButton(identityDOM);
+    await confirmSelectionButton(identityDOM);
```

Comment:
Let's follow the conversation in the front-end channel, but I do believe this screen is not necessary.